store-implementation/federated-store/src/test/java/uk/gov/gchq/gaffer/federatedstore/FederatedGraphStorageTraitsTest.java

```diff
@@ -174,59 +179,79 @@ public void shouldVerifyAssumptionsNoTraitsFound() throws Exception {
+        final Set<StoreTrait> intersectionTraits = accTraits.stream().filter(mapTraits::contains).collect(Collectors.toSet());
 
-        //then
+        // then
         assertEquals(ACCUMULO_TRAITS, accTraits, "This store does not have AccumuloStore Traits");
         assertEquals(MAP_TRAITS, mapTraits, "This store does not have MapStore Traits");
         assertNotEquals(accTraits, mapTraits, "Test stores cannot have same traits");
         assertEquals(10, accTraits.size(), "Expected AccumuloStore trait size is different");
         assertEquals(8, mapTraits.size(), "Expected MapStore trait size is different");
-        assertEquals(MAP_TRAITS_EXCLUSIVE_OF_ACCUMULO, mapTraitsExclusive, "Expected traits exclusive to MapStore is different");
-        assertEquals(ACCUMULO_TRAITS_EXCLUSIVE_OF_MAP, accTraitsExclusive, "Expected traits exclusive to AccumuloStore is different");
+        assertEquals(MAP_TRAITS_EXCLUSIVE_OF_ACCUMULO, mapTraitsExclusive,
+                "Expected traits exclusive to MapStore is different");
+        assertEquals(ACCUMULO_TRAITS_EXCLUSIVE_OF_MAP, accTraitsExclusive,
+                "Expected traits exclusive to AccumuloStore is different");
         assertEquals(INTERSECTION_TRAITS, intersectionTraits, "Expected intersection of traits is different");
         assertEquals(mapTraits, mapTraitsOperation);
         assertEquals(accTraits, accTraitsOperation);
     }
 
     @Test
     public void shouldVerifyAssumptionsStoreTraitsCurrent() throws Exception {
-        //given
+        // given
         getTraits.setCurrentTraits(true);
-        Set<StoreTrait> mapTraitsIsCurrent = map.getGraph().execute(getTraits, testUser);
-        Set<StoreTrait> accTraitsIsCurrent = acc.getGraph().execute(getTraits, testUser);
-
-        //when
-        Set<StoreTrait> mapTraitsIsCurrentExclusive = mapTraitsIsCurrent.stream().filter(t -> !accTraitsIsCurrent.contains(t)).collect(Collectors.toSet());
-        Set<StoreTrait> accTraitsIsCurrentExclusive = accTraitsIsCurrent.stream().filter(t -> !mapTraitsIsCurrent.contains(t)).collect(Collectors.toSet());
-        Set<StoreTrait> intersectionTraitsIsCurrent = accTraitsIsCurrent.stream().filter(mapTraitsIsCurrent::contains).collect(Collectors.toSet());
-        Set<StoreTrait> mapTraitsIsCurrentIsSubSetOfStoreTraits = mapTraitsIsCurrent.stream().filter(t -> !MAP_TRAITS.contains(t)).collect(Collectors.toSet());
-        Set<StoreTrait> accTraitsIsCurrentIsSubSetOfStoreTraits = accTraitsIsCurrent.stream().filter(t -> !ACCUMULO_TRAITS.contains(t)).collect(Collectors.toSet());
-
-        //then
+        final Set<StoreTrait> mapTraitsIsCurrent = map.getGraph().execute(getTraits, testUser);
+        final Set<StoreTrait> accTraitsIsCurrent = acc.getGraph().execute(getTraits, testUser);
+
+        // when
+        final Set<StoreTrait> mapTraitsIsCurrentExclusive = mapTraitsIsCurrent.stream()
+                .filter(t -> !accTraitsIsCurrent.contains(t)).collect(Collectors.toSet());
```

Comment:
👎  You've inserted a lot of line wrap which makes spot the difference harder to see for these tests in future.
```
xxxxx xxxx xx MapTraits xxxx xxx x xx
xxxxx xxxx xx AccTraits xxxx xxx x xx
```